app/gui/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -213,22 +200,36 @@ impl View {
     pub fn new(scene: Scene) -> Self {
         let display_object = display::object::Instance::new();
         let background = background::View::new();
-        let overlay = overlay::View::new();
+        let overlay = Rectangle::default().build(|r| {
+            r.set_color(INVISIBLE_HOVER_COLOR).set_border_color(INVISIBLE_HOVER_COLOR);
+        });
+        let resize_grip = Rectangle::default().build(|r| {
+            r.set_color(INVISIBLE_HOVER_COLOR).set_border_color(INVISIBLE_HOVER_COLOR);
+        });
```

Comment:
Looking through the code, I have no idea how it works: we have only one rectangle for resize grip, which somehow covers both axes, while being also a child of overlay (so technically, it's _over_ overlay).

Reply:
I added the comment explaining it to the definition of `resize_grip` field. So resize grip is below overlay, and has a slight offset to the bottom right, so that it "peeks" from under the overlay. The ordering is done through partition layers, see below.